src/sctools/platform.py

```diff
@@ -418,6 +420,52 @@ def merge_count_matrices(cls, args: Iterable[str]=None) -> int:
         count_matrix.save(args.output_stem)
 
         return 0
+    
+    def group_qc_outputs():
```

Comment:
I would make this `group_picard_qc_metrics` -- it won't work for other metrics in this package.